### PR TITLE
CASSANDRA-18230 CEP 20: DDM cleanup language, rearrange code examples.

### PR DESCRIPTION
**Pull request by polandll**
Authored by Lorina Poland (polandll); Reviewed by ?? for CASSANDRA-18230.

Thanks for sending a pull request! Here are some tips if you're new here:
 
 * Ensure you have added or run the [appropriate tests](https://cassandra.apache.org/_/development/testing.html) for your PR.
 * Be sure to keep the PR description updated to reflect all changes.
 * Write your PR title to summarize what this PR proposes.
 * If possible, provide a concise example to reproduce the issue for a faster review.
 * Read our [contributor guidelines](https://cassandra.apache.org/_/development/index.html)
 * If you're making a documentation change, see our [guide to documentation contribution](https://cassandra.apache.org/_/development/documentation.html)
 
Commit messages should follow the following format:

```
<One sentence description, usually Jira title or CHANGES.txt summary>

<Optional lengthier description (context on patch)>

patch by <Authors>; reviewed by <Reviewers> for CASSANDRA-#####

Co-authored-by: Name1 <email1>
Co-authored-by: Name2 <email2>

```

The [Cassandra Jira](https://issues.apache.org/jira/projects/CASSANDRA/issues/)

### CODE REVIEW

File: doc/modules/cassandra/nav.adoc [hunk 1]
```diff
@@ -61,7 +62,7 @@
 *** xref:cassandra:developing/cql/triggers.adoc[Triggers]
 *** xref:cassandra:developing/cql/appendices.adoc[Appendices]
 *** xref:cassandra:developing/cql/changes.adoc[Changes]
-*** xref:cassandra:developing/cql/SASI.adoc[SASI]
+// *** xref:cassandra:developing/cql/SASI.adoc[SASI]
```
Comment (smiklosovic) [inline doc/modules/cassandra/nav.adoc:65]: why?

**Comment by smiklosovic** [comment]
@polandll 

I see you have commented out SASI https://github.com/apache/cassandra/pull/3090/files#r1489145896

I dont think this change has anything to do with the patch and I would just move it back. We might put a note into SASI that a user should consider using SAI instead but I dont think that this PR is a good vehicle for that. If we are on the same page here, I ll merge the PR into trunk (minus sasi change).

**Comment by polandll** [comment]
Oh geez, this is what happens when you get pulled to another task before truly finishing the first. I have refactored the SASI docs into its own section in developing/cql/indexing/sasi. In another repo. And I didn't finish the task of pulling that writing over to this repo. I will just add the original back in, as you say, for this ticket, and open another PR for the indexing stuff. Thanks for the catch!

File: doc/modules/cassandra/nav.adoc (hunk 1 above)
Comment (polandll) [reply doc/modules/cassandra/nav.adoc:65]: See my comment below.

**Comment by smiklosovic** [comment]
@polandll no worries, I'll take care of that. Thanks for the PR!

**Comment by polandll** [comment]
Yes, this can be merged to trunk, 5.1, 5.0.

**Comment by smiklosovic** [comment]
merged here https://github.com/apache/cassandra/commit/0f2da96c6c617bb08c15303eb8b3cea3a55daf37

**Pull request closed without merging**